### ISSUE TITLE
Provide some closure to the player on character death!

### ISSUE THREAD

**Issue by skillsreactor** [skillsreactor/rpg-learning-example#10]
It's very unsatisfying to die, and we don't have an opportunity later to display the stats because the character save is gone. Let's give ourselves some closure by seeing our character stats one last time.

## User Story
```
AS a player
I WANT to see the final stats of my character when they die
SO THAT I can feel a sense of closure and marvel at my accomplishments one last time.
```

## Acceptance Criteria
```
GIVEN my character has met their fate
WHEN I receive the message notifying me of the outcome
THEN I also see their printed stats.
```

**Comment by Oliverjacobb** [comment]
@skillsreactor see PR below for this... 

https://github.com/skillsreactor/rpg-learning-example/pull/11

**Comment by skillsreactor** [comment]
Closed by #11
